File: cpp/film.cpp
#include <iostream>
#include <string>

using namespace std;

// class Film buat nyimpen data film
class Film{
    private:
        // 4 atribut film
        string idFilm;
        string judul;
        string genre;
        string gambar;

    public:
        // constructor buat mengisi data film
        Film(string id, string nama, string jenis, string foto){
            setIdFilm(id);
            setJudul(nama);
            setGenre(jenis);
            setGambar(foto);
        }

        // setter untuk mengubah data
        void setIdFilm(string id){
            idFilm = id;
        }

        void setJudul(string nama){
            judul = nama;
        }

        void setGenre(string jenis){
            genre = jenis;
        }

        void setGambar(string foto){
            gambar = foto;
        }

        // getter untuk mengambil data
        string getIdFilm(){
            return idFilm;
        }

        string getJudul(){
            return judul;
        }

        string getGenre(){
            return genre;
        }

        string getGambar(){
            return gambar;
        }

        // method untuk menampilkan data film
        void tampilkanData(){
            cout << "ID Film : " << getIdFilm() << endl;
            cout << "Judul   : " << getJudul() << endl;
            cout << "Genre   : " << getGenre() << endl;
            cout << "Gambar  : " << getGambar() << endl;
        }
};

File: java/film.java
// class Film buat nyimpen data film
public class Film {
    // 4 atribut film
    private String idFilm;
    private String judul;
    private String genre;
    private String gambar;

    // constructor buat mengisi data awal film
    public Film(String id, String nama, String jenis, String foto){
        setIdFilm(id);
        setJudul(nama);
        setGenre(jenis);
        setGambar(foto);
    }

    // setter untuk mengubah data
    public void setIdFilm(String id){
        idFilm = id;
    }

    public void setJudul(String nama){
        judul = nama;
    }

    public void setGenre(String jenis){
        genre = jenis;
    }

    public void setGambar(String foto){
        gambar = foto;
    }

    // getter untuk mengambil data
    public String getIdFilm(){
        return idFilm;
    }

    public String getJudul(){
        return judul;
    }

    public String getGenre(){
        return genre;
    }

    public String getGambar(){
        return gambar;
    }

    // method untuk menampilkan data film
    public void tampilkanData(){
        System.out.println("ID Film : " + getIdFilm());
        System.out.println("Judul   : " + getJudul());
        System.out.println("Genre   : " + getGenre());
        System.out.println("Gambar  : " + getGambar());
    }
}